src/lib.rs

```diff
@@ -37,12 +38,13 @@ pub fn get_version() -> String {
 #[pymodule]
 pub fn kornia_rs(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add("__version__", get_version())?;
+    m.add_function(wrap_pyfunction!(read_image_jpeg, m)?)?;
+    m.add_function(wrap_pyfunction!(write_image_jpeg, m)?)?;
     m.add_function(wrap_pyfunction!(read_image_rs, m)?)?;
-    m.add_function(wrap_pyfunction!(cvtensor_to_dlpack, m)?)?;
     m.add_class::<tensor::cv::Tensor>()?;
-
-    #[cfg(feature = "libjpeg-turbo")]
-    m.add_function(wrap_pyfunction!(read_image_jpeg, m)?)?;
+    m.add_class::<ImageSize>()?;
```

Comment:
Will be useful for something in the outside?

Reply:
for now is only to handle the result from header. But i have also in mind to wrap somehow image class that could use this. We need to iterate yet on that api